google-cloud-spanner/src/main/java/com/google/cloud/spanner/DatabaseClientImpl.java

```diff
@@ -37,24 +37,38 @@ class DatabaseClientImpl implements DatabaseClient {
   @VisibleForTesting final SessionPool pool;
   @VisibleForTesting final MultiplexedSessionDatabaseClient multiplexedSessionDatabaseClient;
 
+  final boolean useMultiplexedSessionForBlindWrite;
```

Comment:
Do we need this to be a separate option? Or could we just say 'if multiplexed sessions are enabled, then we use it for both read-only transactions and single-write transactions'?

Reply:
Have removed this flag.